Didi/update hovercell (#910)
telerik/maui-docs: controls/datagrid/cells/mouse-hover-cell.md

You can specify the style for the cells and rows when the mouse is over by using the `MouseHoverStyle` property (of type `DataGridBorderStyle`) and applying the `BackgroundColor`, `BorderColor`, and `BorderThickness` properties.

## Example

> Here is an example how to get the mouse hovered cell and how to apply style to the hovered row.

Reviewer: ```suggestion
Here is an example of how to get the mouse-hovered cell and how to apply styles to the hovered row.
```

**1.** Define sample data:

<snippet id='datagrid-persondetails' />

> **2.** Define sample `ViewModel`:

Reviewer: ```suggestion
**2.** Define a sample `ViewModel`:
```

<snippet id='datagrid-reordering-viewmodel' />

**3.** Define the DataGrid in XAML:

<snippet id='datagrid-mousehovercell' />

> **4.** Define the `MouseHoverStyle` in page;s resources:

Reviewer: ```suggestion
**4.** Define the `MouseHoverStyle` in page's resources:
```
